docusaurus/docs/key-concepts/ui-extensions.md

```diff
@@ -0,0 +1,50 @@
+---
+id: ui-extensions
+title: UI extensions
+description: Learn how to add links and trigger actions from the Grafana user interface by using UI Extensions in app plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - links
+  - extensions
+  - app plugins
+sidebar_position: 60
+---
+
+Use UI extensions to contribute new actions and functionality to the core Grafana UI and other app plugins.
+
+## Definition of terms
+
+Before we go into details we need to cover the two major concepts in the UI extensions framework:
+
+1. `Extension point` - A place in the UI where plugins can contribute new functionality to the end user. The Grafana UI exposes extension points and app plugins can also define their own extension points.
+2. `Extension` - New functionality, registered by an app plugin, appearing at an extension point. For example, an extension could provide a navigational link to bring users to a particular view, it could open a modal menu allowing the user to configure an action to take from within their current context (for instance, to create a SLO), or it could trigger background tasks.
+
+![Panel menu showing available extensions](/img/ui-extensions-menu.png)
+
+In the example above, there is one extension point with three registered extensions. This highlights one of the benefits of using UI extensions. Once you have added an extension point to your UI you can extend it multiple times by multiple plugins.
+
+## Why should I add an extension point?
+
+App plugins can provide custom pages in the Grafana UI, often highly contextualized to a particular service or task to enable users to be productive. Grafana is a feature-rich observability platform and has an extensive ecosystem of plugins, allowing users to monitor and act upon a wide set of data.
+
+Extension points facilitate the breaking down of silos between individual views, allowing users to quickly leverage data from their current context to take relevant actions. By providing extension points within your app, relevant extensions can easily offer new capabilities to your shared users.
+
+Add an extension point to your UI to give benefits such as these:
+
+- Define the UI extension point once to enable multiple plugins to extend your UI with new functionality. You don't need any additional effort to provide functionality from more plugins in your UI.
+- Clean separation of concerns. Your application doesn't need to know anything about the plugin extending your UI.
+- Integration build for change. Since your application doesn't know anything about the internal workings of the plugin extending your UI, they are free to change their APIs without the risk of breaking the extension point UI.
```

Comment:
Typo?
```suggestion
- Integration built for change. Since your application doesn't know anything about the internal workings of the plugin extending your UI, they are free to change their APIs without the risk of breaking the extension point UI.
```